lib/src/bottom_sheet_route.dart

```diff
@@ -32,17 +32,22 @@ class _ModalBottomSheet<T> extends StatefulWidget {
 }
 
 class _ModalBottomSheetState<T> extends State<_ModalBottomSheet<T>> {
-  String _getRouteLabel(MaterialLocalizations localizations) {
-    final platform = Theme.of(context).platform;
-    switch (platform) {
-      case TargetPlatform.iOS:
-        return '';
-      case TargetPlatform.android:
-      case TargetPlatform.fuchsia:
-        return localizations.dialogLabel;
-        break;
+  String _getRouteLabel() {
+    if (Theme.of(context, shadowThemeOnly: true) == null) {
```

Comment:
Nice! Maybe instead of checking `if(Theme.of(context, shadowThemeOnly: true) == null)`
do this in line 39
`final platform = Theme.of(context)?.platform ?? defaultTargetPlatform;`

Reply:
Same here, I did not test the combination:
```
Theme(
   data: ThemeData(),
   child: CupertinoApp(...),
)
```

I tested your suggestion and works well. Maybe we should a comment, that using this package with an CupertinoApp or WidgetApp and testing on an Android Emulator requires to set `debugDefaultTargetPlatformOverride = TargetPlatform.iOS;`.

Reply:
Yeah, or instead of the assert just check ( Localizations.of<MaterialLocalizations>(context, MaterialLocalizations) != null); when TargetPlatform is Android. 

Overriding the default Target Platform is advise against and it is built only for testing.

Reply:
The check would be a nice way I think. But I have a question, should we use `DefaultMaterialLocalizations().dialogLabel` as default value ifLocalizations are not available ?
So the implemetation would be as follows:
```
String _getRouteLabel() {
    final platform = Theme.of(context)?.platform ?? defaultTargetPlatform;
    switch (platform) {
      case TargetPlatform.iOS:
        return '';
      case TargetPlatform.android:
      case TargetPlatform.fuchsia:
        if (Localizations.of(context, MaterialLocalizations) != null) {
          return MaterialLocalizations.of(context).dialogLabel;
        } else {
          return DefaultMaterialLocalizations().dialogLabel;
        }
    }
    return null;
  }
```

Reply:
Sounds good to me 👍